### PR TITLE
Add support for custom prettifiers on the metadata

### PR DESCRIPTION
**Pull request by airhorns**
So folks can prettify the pid, logger name, or hostname as they see fit!

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by mcollina** [comment]
CI is failing

File: lib/utils.js [hunk 1]
```diff
@@ -285,33 +285,36 @@ function prettifyMessage ({ log, messageFormat, messageKey = MESSAGE_KEY, colori
  * @returns {undefined|string} If no metadata is found then `undefined` is
  * returned. Otherwise, a string of prettified metadata is returned.
  */
-function prettifyMetadata ({ log }) {
+function prettifyMetadata ({ log, prettifiers = {} }) {
```
Comment (jsumners) [inline lib/utils.js:288]: Please document this in the docblock.

**Review by jsumners — CHANGES_REQUESTED** [review]
Please update the documentation showing how to use this feature.

**Comment by airhorns** [comment]
Updated with docs and the coverage target

**Review by mcollina — APPROVED** [review]
lgtm

File: lib/utils.js [hunk 2]
```diff
@@ -279,39 +279,42 @@ function prettifyMessage ({ log, messageFormat, messageKey = MESSAGE_KEY, colori
  * the fields it finds.
  *
  * @param {object} input
- * @param {object} input.log The log that may or may not contain metadata to
- * be prettified.
+ * @param {object} input.log The log that may or may not contain metadata to be prettified
```
Comment (jsumners) [inline lib/utils.js:282]: Please re-wrap this as it was before and restore the punctuation.

**Review by jsumners — CHANGES_REQUESTED** [review]

**Review by jsumners — APPROVED** [review]
Looks good to me.

**Pull request merged**